packages/worker/src/transcribe.ts

```diff
@@ -172,27 +176,45 @@ const runTranscription = async (
 	}
 };
 
+const getLanguageCode = async (
+	whisperBaseParams: WhisperBaseParams,
+	whisperX: boolean,
+): Promise<LanguageCode> => {
+	if (whisperX) {
+		return Promise.resolve('auto');
+	}
+	const dlParams = whisperParams(true, whisperBaseParams.wavPath);
+	const { metadata } = await runWhisper(whisperBaseParams, dlParams);
```

Comment:
this was the case before your change, but are we running whisper twice (or thrice if we're translating)? once here and once in `runTranscription`?

Reply:
might be worth adding a comment to explain